airflow/www/views.py

```diff
@@ -1724,6 +1742,68 @@ def gantt(self):
             root=root,
         )
 
+    @expose('/extra_links')
+    @login_required
+    @wwwutils.action_logging
+    def extra_links(self):
```

Comment:
Curious about why not bootstrapping the data into the page instead of getting it async. Are you expecting expensive/slow calls here?

Reply:
So the reason I do it asynchronous is because the tree view of days loads all the execution dates at once. A relatively cheap remote call locally (let's say .2 seconds) becomes an additional 5 seconds to load the tree view (because I think I remember it only loads 25 dates at a time).

Reply:
Sor turns out my math is wrong, it would be 25 *.2 * `X` where X is the number of operators that have to render that link. So it could easily be > 5 seconds.